.gitignore

```diff
@@ -9,3 +9,6 @@ node_modules/
 
 # build
 realm-cli*
+
+# VSCode
+.vscode
```

Comment:
happy to remove this from this PR if preferred

Reply:
there are probably enough people using `vscode` where this is helpful... feel free to keep as is!